lib/intMap.js

```diff
@@ -0,0 +1,51 @@
+var hasOwn = Object.prototype.hasOwnProperty;
+
+/**
+ * A very simple polyfill for Map with limited functionality
+ * that only works for integer keys.
+ */
+function IntMap() {
+    this.map = {};
```

Comment:
I didn't do it because I'm using `Object.prototype.hasOwnProperty`. Is there a perf improvement to doing `Object.create(null)`?